pkg/gofr/service/health_config.go

```diff
@@ -4,20 +4,28 @@ import "context"
 
 type HealthConfig struct {
 	HealthEndpoint string
+	Timeout        int
 }
 
 func (h *HealthConfig) AddOption(svc HTTP) HTTP {
+	// if timeout is not provided we set the timeout to 5 sec by default.
```

Comment:
Magic number in comment, the value may change and this comment will be outdated


```suggestion
	// if timeout is not provided we set the timeout to a convenient default value.
```